conf/applicationContext-security-ldap.properties

```diff
@@ -0,0 +1,34 @@
+#Provided armedia dev ldap details
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.